sys/kern/subr_intr.c

```diff
@@ -1552,14 +1595,18 @@ dosoftints(void)
 void
 intr_pic_init_secondary(void)
 {
-
 	/*
-	 * QQQ: Only root PIC is aware of other CPUs ???
+	 * QQQ: Only root PICs are aware of other CPUs ???
 	 */
-	KASSERT(intr_irq_root_dev != NULL, ("%s: no root attached", __func__));
-
+	uint32_t rootnum;
+	device_t dev;
```

Comment:
I would to have these declarations above the comment.  Then FreeBSD style would be blank line between the two.

Reply:
(blank line between the declarations and the comment)

Reply:
done